address-controller/src/main/java/io/enmasse/controller/ControllerHelper.java

```diff
@@ -165,14 +157,12 @@ private StandardResources createResourceList(AddressSpace addressSpace) {
             }
 
             // Step 3: Create missing secrets if not specified
-            Set<String> secretsToGenerate = new HashSet<>();
             for (String service : availableServices) {
                 String secretName = getSecretName(service);
 
-                    // TODO: Remove console clause when it supports https
+                // TODO: Remove console clause when it supports https
```

Comment:
Do we have an issue covering this TODO?  If we do we should remove the TODO.. if we don't then we should create an issue... and then remove the TODO :-)

Reply:
https://github.com/EnMasseProject/enmasse/issues/402 :) Removing the TODO